core/src/main/java/org/apache/iceberg/V1Metadata.java

```diff
@@ -350,6 +351,8 @@ public Object get(int pos) {
           return wrapped.keyMetadata();
         case 12:
           return wrapped.splitOffsets();
+        case 13:
+          return specId();
```

Comment:
Required to keep tests passing, when fields are not named in the schema avro makes up it's own field name which breaks some tests.

Reply:
This code is only exercised because ManifestEntryTable (possibly incorrectly) only uses V1Metdata methods to try to read manifests.